### ISSUE TITLE
Dragonfly server crash

### ISSUE THREAD

**Issue by lexikiq** [paulevsGitch/BetterEnd#7]
Using latest version from curseforge (0.3.2-alpha)

Occurred a couple times while exploring a Stonebrick Stronghold from [Repurposed Structures](https://www.curseforge.com/minecraft/mc-mods/repurposed-structures-fabric/), although it was just a normal vanilla stronghold placed in the end, so I don't believe it to be anything specific to that mod causing the crash.

```
[13:26:13] [Server thread/ERROR]: Encountered an unexpected exception
net.minecraft.class_148: Ticking entity
        at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:872) ~[intermediary-server.jar:?]
        at net.minecraft.class_3176.method_3813(class_3176.java:303) ~[intermediary-server.jar:?]
        at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:808) ~[intermediary-server.jar:?]
        at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:667) ~[intermediary-server.jar:?]
        at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:254) ~[intermediary-server.jar:?]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_262]
Caused by: java.lang.ArrayIndexOutOfBoundsException: -1
        at net.minecraft.class_14.method_59(class_14.java:739) ~[intermediary-server.jar:?]
        at net.minecraft.class_6.method_25(class_6.java:310) ~[intermediary-server.jar:?]
        at net.minecraft.class_14.method_64(class_14.java:410) ~[intermediary-server.jar:?]
        at net.minecraft.class_6.method_17(class_6.java:257) ~[intermediary-server.jar:?]
        at net.minecraft.class_6.method_9(class_6.java:321) ~[intermediary-server.jar:?]
        at net.minecraft.class_6.method_13(class_6.java:228) ~[intermediary-server.jar:?]
        at net.minecraft.class_6.method_18(class_6.java:107) ~[intermediary-server.jar:?]
        at net.minecraft.class_13.method_54(class_13.java:97) ~[intermediary-server.jar:?]
        at net.minecraft.class_13.method_52(class_13.java:45) ~[intermediary-server.jar:?]
        at net.minecraft.class_1408.method_18416(class_1408.java:195) ~[intermediary-server.jar:?]
        at net.minecraft.class_1408.method_6348(class_1408.java:153) ~[intermediary-server.jar:?]
        at ru.betterend.entity.EntityDragonfly$WanderAroundGoal.method_6269(EntityDragonfly.java:146) ~[better-end-0.3.2-alpha.jar:?]
        at net.minecraft.class_4135.method_6269(class_4135.java:41) ~[intermediary-server.jar:?]
        at net.minecraft.class_1355.startGoals(class_1355.java:636) ~[intermediary-server.jar:?]
        at net.minecraft.class_1355.updateGoalStates(class_1355.java:573) ~[intermediary-server.jar:?]
        at net.minecraft.class_1355.method_6275(class_1355.java:555) ~[intermediary-server.jar:?]
        at net.minecraft.class_1308.method_6023(class_1308.java:736) ~[intermediary-server.jar:?]
        at net.minecraft.class_1309.method_6007(class_1309.java:2508) ~[intermediary-server.jar:?]
        at net.minecraft.class_1308.method_6007(class_1308.java:542) ~[intermediary-server.jar:?]
        at net.minecraft.class_1296.method_6007(class_1296.java:127) ~[intermediary-server.jar:?]
        at net.minecraft.class_1429.method_6007(class_1429.java:54) ~[intermediary-server.jar:?]
        at net.minecraft.class_1309.method_5773(class_1309.java:2254) ~[intermediary-server.jar:?]
        at net.minecraft.class_1308.method_5773(class_1308.java:342) ~[intermediary-server.jar:?]
        at net.minecraft.class_3218.method_18762(class_3218.java:616) ~[intermediary-server.jar:?]
        at net.minecraft.class_1937.method_18472(class_1937.java:561) ~[intermediary-server.jar:?]
        at net.minecraft.class_3218.method_18765(class_3218.java:406) ~[intermediary-server.jar:?]
        at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:868) ~[intermediary-server.jar:?]
```

**Comment by paulevsGitch** [comment]
Thanks for the report, v 0.4.0 will fix that (and will fix other server-side issues)

**Comment by paulevsGitch** [comment]
Looks like the reason is in empty path in AI, fixed that

**Comment by paulevsGitch** [comment]
Fixed in 0.4.0

**Comment by lexikiq** [comment]
thank you! awesome mod, by the way. the biomes are gorgeous :)
